data/events/2024-denver.yml

```
@@ -34,8 +34,8 @@ location: "Mile High Station and Iron Works" # Defaults to city, but you can mak
 location_address: "2027 W Colfax Ave, Denver, CO 80204" #Optional - use the street address of your venue. This will show up on the welcome page if set.
 
 nav_elements: # List of pages you want to show up in the navigation of your page.
- # - name: propose
- # - name: location
+ - name: propose
+ - name: location
```

I think the errors are coming from the space here. The `propose` and `sponsor` items must align otherwise yaml gets cranky
```suggestion
  - name: propose
  - name: location
```

---

Good catch, thanks @glasnt! Just updated the PR